examples/fast_fourier_transformation/fft.cpp

```diff
@@ -0,0 +1,76 @@
+//  Copyright (c) 2022 Deepak Suresh
+
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#include <hpx/hpx.hpp>
+#include <hpx/hpx_init.hpp>
+
+#include <iostream>
+#include <math.h>
+#include <vector>
+
+// For storing complex values of nth roots of unity
+using cd = std::complex<double>;
+
+// Function for fast fourier transformation
+std::vector<cd> fft(const std::vector<cd>& a)
```

Comment:
We prefer using 'east const' over 'west const' notations. Could you adapt your code, please?